anitya/db/models.py

```diff
@@ -335,10 +336,15 @@ def create_version_objects(self, versions):
         versions = sorted(
             [
                 version_class(
-                    version=version,
+                    version=version if isinstance(version, str) else version["version"],
```

Comment:
Why this change?

Reply:
Oops, I seem to have missed to commit the code which actually passes around cursors with the versions here. Hang on.